sp800-63b/sec6_lifecycle.md

```diff
@@ -82,3 +82,32 @@ The CSP SHALL require subscribers to surrender or prove destruction of any physi
 
 Further requirements on the termination of PIV credentials are found in [[FIPS 201]](#FIPS201).
 
+### 6.5. Derived Authenticators
+
+IAL and AAL as separate ordinals allows organizations to issue any type of authenticator they want regardless of how or if an individual is identity proofed.  This is one of the benefits of the separation - an organization can issue strong credentials regardless of how or if identity proofing is performed.  However, it may be beneficial for organizations to issue authenticators based on the individual having an authenticator bound to a credential that has an IAL of 2 or 3.
```

Comment:
"The expression of IAL and AAL as separate ordinals..."

individual -> subject

"issue authenticators suitable for use by subscribers having a credential at IAL 2 or 3."